pdr_backend/accuracy/app.py

```diff
@@ -247,12 +248,33 @@ def calculate_timeframe_timestamps(
         # else timedelta(days=1)
     )
     start_ts = UnixTimeS(int((datetime.utcnow() - time_delta).timestamp()))
-
     return start_ts, end_ts
 
 
 @enforce_types
-def save_statistics_to_file():
+def fetch_statistics_using_ETL():
+    # return
+    ppss = PPSS(
+        yaml_filename="./ppss.yaml",
+        network="sapphire-mainnet",
+        nested_override_args=None,
+    )
+
+    gql_data_factory = GQLDataFactory(ppss)
+    etl = ETL(ppss, gql_data_factory)
+    while True:
+        etl.do_etl()
+        threading.Event().wait(300)  # Wait for 5 minutes (300 seconds)
+
+
+@enforce_types
+def filter_func(seconds_per_epoch: int) -> Callable[[Any], bool]:
+    # filter out all the items that have seconds_per_epoch different that the specified parameter
+    return lambda item: int(item.get("seconds_per_epoch", 0)) == seconds_per_epoch
```

Comment:
`return int(item.get("seconds_per_epoch", 0)) == seconds_per_epoch` is enough.

lambda is a way to define anonymous functions but this function is named ("filter_func"). Using a lambda as an alias to a named function is moot.